packages/airnode-deployer/terraform/airnode/aws/main.tf

```diff
@@ -39,7 +39,7 @@ module "processProviderRequests" {
   handler            = "index.processProviderRequests"
   source_dir         = var.handler_dir
   memory_size        = 768
-  timeout            = 32
+  timeout            = 6
```

Comment:
This feels too little to me. I would not take the stress test as the absolute guideline for the timeouts (it's still a limited test scenario). I'd maybe go with 10 here (so that it has a little more time)

Reply:
All good, I agree that 10 is safer and certainly would be necessary for slow nodes.
This timing still gives > 250 requests per cycle.